templates/advantage/index.html

```diff
@@ -58,7 +58,170 @@ <h2>Your paid subscriptions</h2>
     </section>
 
     <section class="p-strip u-no-padding--top is-shallow">
-      <div class="row">
+      <div class="row u-hide--medium u-hide--large">
+        <div class="col-12">
+          {% for account in enterprise_contracts %}
+          {% set outer_loop = loop %}
+          <h4>{{ account }}</h4>
+          {% for contract in enterprise_contracts[account] %}
+          {% set expired_renewable = contract['renewal'] and contract['contractInfo']['daysTillExpiry'] < 0 %}
+          <div class="p-card">
+            <h5 class="p-card__title u-sv1 u-no-margin--bottom">
+              {% if contract["contractInfo"]["id"] == new_subscription_id %}
+                {% if "daysTillExpiry" in contract["contractInfo"] and contract["contractInfo"]["daysTillExpiry"] > 0 %}
+                  <div class="p-label--new">New</div>
+                {% endif %}
+              {% endif %}
+              {{ contract['contractInfo']['name'] }}
+            </h5>
+            <hr class="u-sv1">
+            <div class="row p-card__content">
+              {% if new_subscription_id == contract['contractInfo']['id'] %}
+              <h4>Welcome to your new subscription.</h4>
+              <p>To get started, first attach each machine:</p>
+              <p>To attach a machine: <code>sudo ua attach {{ contract['token'] }}</code></p>
+              <p>To detach a machine: <code>sudo ua detach</code></p>
+              <p>
+                Once a machine is attached, you can configure its services. Visit the <a href="http://manpages.ubuntu.com/manpages/focal/en/man1/ua.1.html" class="p-link--external">UA manpages</a> for details.
+              </p>
+              {% else %}
+              <div class="col-12 u-align--center u-sv3">
+                To attach a machine: <br /><code>sudo ua attach {{ contract['token'] }}</code>
+              </div>
+              {% endif %}
+            </div>
```

Comment:
In the large-screen view:
- the attach instruction is hidden by default (except for a new subscription)
- the detach instruction is hidden by default (ditto).

But in this mobile view:
- the attach instruction is _always_ shown, with no ability to hide it
- the detach instruction is _never_ shown, with no ability to show it!

Reply:
Would you prefer all info to be hidden by default as the large-screen view does?

Reply:
By default, yes, with the ability to reveal it. This would involve “Name” switching from a header to an expandable row, and the “Attached” row becoming expandable too (both shown in the wireframe).